app/helpers/ontology_version_helper.rb

```diff
@@ -0,0 +1,32 @@
+module OntologyVersionHelper
+  def failed?(ontology_version)
+    ontology_version.state == 'failed'
+  end
+
+  def latest_version?(ontology_version)
+    ontology_version.ontology.versions.last == ontology_version
```

Comment:
I'm not sure if it's better to check it this way (was using the iteration variable from the view before). Probably more robust, but definitely slower.